models/_generate_layers.py

```diff
@@ -496,6 +497,50 @@ def collect_layers():
   return ls
 
 
+def get_super_call_params(layer: LayerBase) -> str:
```

Comment:
Consistently to the comment above, this function should be called sth like `get_super_call_kwargs_repr` or so.